subsys/mpsl/cx/Kconfig

```diff
@@ -77,19 +77,18 @@ config MPSL_CX_3WIRE
 	  implementation for co-located 2.4 GHz radios.
 
 config MPSL_CX_BT_1WIRE
-	depends on SOC_COMPATIBLE_NRF52X
-	select NRFX_GPIOTE
-	select GPIO
-	bool "Bluetooth Radio 1-Wire Coexistence"
+	select DEPRECATED
+	select NRFX_GPIOTE if !MPSL_CX_PIN_FORWARDER
+	select GPIO if !MPSL_CX_PIN_FORWARDER
+	bool "Bluetooth Radio 1-Wire Coexistence [DEPRECATED]"
 	help
 	  Radio Coexistence interface implementation using a simple 1-wire PTA
```

Comment:
Maybe this could mention it's been replaced by `CONFIG_MPSL_CX_1WIRE`?

Reply:
Under help, right?